lib/src/components/NxList/NxListActions.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {NxListActionProps, nxListActionPropTypes} from './types';
+
+const NxListAction = (props: NxListActionProps) => {
```

Comment:
This was more succinctly and more powerfully defined as a convenience component.  I say more powerfully because this implementation doesn't accept and extra attrs or class names.

Reply:
> This was more succinctly and more powerfully defined as a convenience component. I say more powerfully because this implementation doesn't accept and extra attrs or class names.

Added extra `attrs` and support for `className`.
6692c10b151969d026663c8b5331b550cbb5bf31